network/msgCompressor.go

```diff
@@ -219,7 +220,11 @@ func (c *wsPeerMsgCodec) decompress(tag protocol.Tag, data []byte) ([]byte, erro
 		if c.avdec.enabled {
 			res, err := c.avdec.convert(data)
 			if err != nil {
-				c.log.Warnf("peer %s vote decompress error: %v", c.origin, err)
```

Comment:
I vote for disconnecting from nodes who advertised vpack compression but fail to fulfill it